src/views/apply/dashboard.html

```diff
@@ -58,81 +62,149 @@
+        <a href="{{ internationalTravelForm }}" target="_blank">Travel Details (international hackers)</a>
+      </p>
     </section>
   {% endif %}
 
   {# An extra details section for invited, non-declined hackers #}
-  {% if rsvpStatus == statuses.rsvp.COMPLETE_YES or rsvpStatus == statuses.rsvp.INCOMPLETE %}
+
+  {% if responseStatus == statuses.response.INVITED and rsvpStatus != statuses.rsvp.COMPLETE_NO %}
     {# Date and Time #}
     <section class="dashboard-section">
       <h3>Date and Time</h3>
-      Friday 28th Jan 2017 - Saturday 29th Jan <br>
-      (Times TBC)
+      <p>
+        Friday 28th Jan 2017 (Registration opens at 9am) - Saturday 29th Jan (Closing ceremony finishes at 6pm) <br>
+        (Times subject to change)
+      </p>
     </section>
 
     {# Location #}
     <section class="dashboard-section">
       <h3>Location</h3>
-      Cambridge Corn Exchange,<br>
-      2 Wheeler St,<br>
-      Cambridge<br>
-      CB2 3QB<br>
+      <p>
+        Cambridge Corn Exchange,<br>
+        2 Wheeler St,<br>
+        Cambridge<br>
+        CB2 3QB<br>
+      </p>
       <a href="https://goo.gl/maps/6TTeXNnD7Jp" target="_blank">View on Google Maps &rarr;</a>
     </section>
 
     {# Travel Reimbursements #}
     <section class="dashboard-section">
       <h3>Travel Cost Reimbursements</h3>
-      We will be able to reimburse a small amount of your travel costs. You can apply for a travel reimbursement of 50% of your costs, up to a maximum of £60. Make sure you keep your receipts!
-      <br>
-      <strong>Note:</strong> We won’t be reimbursing parking costs or cost of travel from locations that we provide coaches for (more info and sign ups for coaches coming soon!), nor will we be reimbursing any students who study locally. We ask that you be responsible about claiming travel reimbursements and only do so if appropriate — we have limited resources!
+      <p>
+        We will be able to reimburse a small amount of your travel costs. The amount you will be able to claim depends on where you're coming from:<br>
+        You can apply for up to £30 for distances as far as london,<br>
+        Up to £50 for distances as far as Manchester,<br>
+        And £70 for distances further than Manchester (including international)<br>
+      </p>
+      <p>We will send more details on travel reimbursements, and we may be processing them at the event. Make sure you keep your receipts!</p>
+      <p>
+        <strong>Note:</strong> We won’t be reimbursing parking costs or cost of travel if you have indicated that you will be travelling on one of our coaches, nor will we be reimbursing any students who study locally. We ask that you be responsible about claiming travel reimbursements and only do so if appropriate — we reserve the right to deny travel reimbursements.
+      </p>
+      <p>
+        If you haven't already done so, remember to fill in the travel form:<br>
```

Comment:
I think we should be explicit on the purposes of these forms. "UK" is for expression of interest for coaches, "International" is for people who need visa letters. That information isn't relevant for all people, is it?